.make/check-gopath.go

```diff
@@ -34,5 +34,5 @@ func main() {
 		}
 	}
 
-	log.Fatal(fmt.Errorf("Make sure you've checked out your project in GOPATH/src/%s", packageName))
+	log.Fatal(fmt.Errorf("Make sure you've checked out your project in $GOPATH/src/%s", packageName))
```

Comment:
I explicitly left out the `$` because then the message would be wrong on Windows where it should be `%GOPATH%`. Clearly `GOPATH` is wrong on both systems. Maybe this is better:

```go
goPathStr := path.Join("$GOPATH", "src", packageName)
if runtime.GOOS == windows {
    goPathStr := path.Join("%GOPATH%", "src", packageName)
}
log.Fatal(fmt.Errorf("Make sure you've checked out your project in %s", goPathStr)
```